openlane/steps/klayout.py

```diff
@@ -185,6 +185,21 @@ class StreamOut(KLayoutStep):
     id = "KLayout.StreamOut"
     name = "GDSII Stream Out (KLayout)"
 
+    config_vars = KLayoutStep.config_vars + [
+        Variable(
+            "ISOSUB_LAYER",
+            Optional[Tuple[int, int]],
+            "Layer/datatype pair for subcut",
```

Comment:
Comment should probably state not to mess with this unless you know what you're doing